.github/workflows/integration_tests.yml

```diff
@@ -99,9 +99,6 @@ jobs:
     env:
       PLAYWRIGHT_BASE_URL: ${{ github.event.inputs.springfield_service_hostname }}
       CI: true
-      CI_JOB_ID: ${{ github.run_id }}
-      DRIVER: ""
-      LABEL: playwright-tests
```

Comment:
FYI these are being removed because they are not used by the CI, or Playwright here. It's leftovers from functional tests, mainly to configure Selenium, pytest and codecov, which is not relevant for these jobs.